README.md

```diff
@@ -46,6 +46,17 @@ pip install torch~=2.3.0 torch_xla~=2.3.0 https://storage.googleapis.com/pytorch
 
 To use the plugin, set `XLA_REGISTER_INSTALLED_PLUGINS=1` or call `torch_xla.experimental.plugins.use_dynamic_plugins()` in your script.
 
+## Build from source
```

Comment:
I think we already have https://github.com/pytorch/xla/blob/master/CONTRIBUTING.md and mentioned it in the README

Reply:
Thanks, would close this PR